bookkeeper-server/pom.xml

```diff
@@ -35,10 +35,9 @@
       <version>${project.parent.version}</version>
     </dependency>
     <dependency>
-      <groupId>com.google.protobuf</groupId>
-      <artifactId>protobuf-java</artifactId>
-      <version>${protobuf.version}</version>
-      <scope>compile</scope>
+      <groupId>org.apache.bookkeeper</groupId>
```

Comment:
What about project.parent.grouiId?

Reply:
We don't use it for common. We should be consistent IMO.